jvm/src/main/scala/org/apache/spark/ml/rapids/RapidsLogisticRegressionModel.scala

```diff
@@ -50,4 +56,44 @@ class RapidsLogisticRegressionModel(override val uid: String,
     newModel
   }
 
+  override def write: MLWriter =
+    new RapidsLogisticRegressionModel.RapidsLogisticRegressionModelWriter(this)
+}
+
+object RapidsLogisticRegressionModel extends MLReadable[RapidsLogisticRegressionModel] {
+
+  override def read: MLReader[RapidsLogisticRegressionModel] = new RapidsLogisticRegressionModelReader
+
+  override def load(path: String): RapidsLogisticRegressionModel = super.load(path)
+
+  private class RapidsLogisticRegressionModelWriter(instance: RapidsLogisticRegressionModel)
+    extends MLWriter with Logging {
+
+    override protected def saveImpl(path: String): Unit = {
+      val writer = instance.cpuModel.write
+      if (shouldOverwrite) {
+        writer.overwrite()
+      }
+      optionMap.foreach { case (k, v) => writer.option(k, v) }
+      writer.save(path)
+
+      val attributesPath = new Path(path, "attributes").toString
```

Comment:
We should sync up with cpu fallback change PR.   Currently cpu fallback just returns a cpu model, no additional attributes.    Transform PR would be impacted as well.   Not sure that is what we want though.